engine/admin/logging/awslogs.md

```diff
@@ -14,10 +14,22 @@ and Command Line Tools](http://docs.aws.amazon.com/cli/latest/reference/logs/ind
 
 ## Usage
 
-You can configure the default logging driver by passing the `--log-driver`
-option to the Docker daemon:
+To use the `awslogs` driver as the default logging driver, set the `log-driver`
+and `log-opt` keys to appropriate values in the `daemon.json` file, which
+is located in `/etc/docker/` on Linux hosts or
+`C:\ProgramData\docker\config\daemon.json` on Windows Server. The following
+example sets the log driver to `awslogs` and sets the `awslogs-region` option.
```

Comment:
Perhaps we should link to the `daemon.json` reference in the `dockerd` CLI reference